### PR TITLE
Update us.m3u [West Coast]

### PR DESCRIPTION
**Pull request by Central302**
This Patch Will Adding West Coast Feed of BIG4 Networks (These Are Based in California State), (Thank You to Research of Link by @what-a)

### CODE REVIEW

File: streams/us.m3u [hunk 1]
```diff
@@ -1759,4 +1759,12 @@ https://wscvlive-lh.akamaihd.net/i/MIAMI_STREAM6@183618/master.m3u8
 https://wtvjlive-f.akamaihd.net/i/MIAMI_STREAM1@19309/master.m3u8
 #EXTINF:-1 tvg-id="" status="error" user-agent="Mozilla/5.0 Macintosh; Intel Mac OS X 10_14_5 AppleWebKit/537.36 KHTML, like Gecko Chrome/76.0.3809.25 Safari/537.36",Warner TV (720p) [Not 24/7]
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 Macintosh; Intel Mac OS X 10_14_5 AppleWebKit/537.36 KHTML, like Gecko Chrome/76.0.3809.25 Safari/537.36
-https://www.livedoomovie.com/02_WarnerTVHD_720p/chunklist.m3u8
+https://www.livedoomovie.com/02_WarnerTVHD_720p/chunklist.m3u8#EXTM3U
```
Comment (LaneSh4d0w) [inline streams/us.m3u:1762]:
```suggestion
https://www.livedoomovie.com/02_WarnerTVHD_720p/chunklist.m3u8
```

**Comment by LaneSh4d0w** [comment]
CBS KPIX : http://51.161.118.146:8080/ISG03_CBS_SAN_FRANCISCO_CA_KPIX/video.m3u8

**Review by what-a — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Comment by mammasz** [comment]
Hello

**Comment by playxx** [comment]
Anyone has current west coast big 4 feeds? 
Above  CBS KPIX : http://51.161.118.146:8080/ISG03_CBS_SAN_FRANCISCO_CA_KPIX/video.m3u8, etc. 
is not working right.

thanks.

**Comment by LaneSh4d0w** [comment]
> Anyone has current west coast big 4 feeds? Above CBS KPIX : http://51.161.118.146:8080/ISG03_CBS_SAN_FRANCISCO_CA_KPIX/video.m3u8, etc. is not working right.
> 
> thanks.

The server that used to host it closed port 8080, I guess it's over for these links.

**Pull request merged**